src/modules/cpu/kernel/resize.hpp

```diff
@@ -0,0 +1,1109 @@
+#include "rppdefs.h"
+#include "cpu/rpp_cpu_simd.hpp"
+#include "cpu/rpp_cpu_common.hpp"
+
+RppStatus resize_u8_u8_host_tensor(Rpp8u *srcPtr,
+                                   RpptDescPtr srcDescPtr,
+                                   Rpp8u *dstPtr,
+                                   RpptDescPtr dstDescPtr,
+                                   RpptImagePatchPtr dstImgSize,
+                                   RpptROIPtr roiTensorPtrSrc,
+                                   RpptRoiType roiType,
+                                   RppLayoutParams srcLayoutParams)
+{
+    RpptROI roiDefault;
+    RpptROIPtr roiPtrDefault;
+    roiPtrDefault = &roiDefault;
+    roiPtrDefault->xywhROI.xy.x = 0;
+    roiPtrDefault->xywhROI.xy.y = 0;
+    roiPtrDefault->xywhROI.roiWidth = srcDescPtr->w;
+    roiPtrDefault->xywhROI.roiHeight = srcDescPtr->h;
+
+omp_set_dynamic(0);
+#pragma omp parallel for num_threads(dstDescPtr->n)
+    for(int batchCount = 0; batchCount < dstDescPtr->n; batchCount++)
+    {
+        RpptROI roi;
```

Comment:
L26-L52 could be replaced now with someething like 21-L23 here - https://github.com/r-abishek/rpp/blob/ar/opt_resize_bilinear_host/src/modules/cpu/kernel/brightness.hpp#L21-L23